source/particle/world.cc

```diff
@@ -102,15 +97,14 @@ namespace aspect
       if (!output)
         return "";
 
-      particle_timer->enter_section("Write output");
+      TimerOutput::Scope timer_section(*particle_timer,"Particles: Output");
```

Comment:
How about naming the section "Postprocess/Particles/Output"? This sorts it under the "Postprocess" section, and suggests that it is a subset of the time.